etc/README.md

```diff
@@ -11,8 +12,64 @@ The files in this directory may undergo a lot of changes while reth is unstable,
 
 ### Docker Compose
 
-To run Reth, Grafana or Prometheus with Docker Compose, refer to the [docker docs](/book/installation/docker.md#using-docker-compose).
+To run Reth, Grafana or Prometheus with Docker Compose, refer to
+the [docker docs](/book/installation/docker.md#using-docker-compose).
 
-### Import Grafana dashboards
+Thank you for providing that context. I understand now that this text is already part of a subsection. I'll revise it to
+fit better within that structure while still improving clarity and readability.
+
+### Grafana
+
+#### Adding a new metric to Grafana
+
+To set up a new metric in the Reth Grafana dashboard:
+
+1. Add the metric to the codebase following the [metrics section](../docs/design/metrics.md#creating-metrics)
+   documentation.
+
+2. Build the Reth image:
+   ```bash
+   docker build . -t reth:local
+   ```
+   Modify the [docker-compose](docker-compose.yml) file to use your locally built image for the Reth service.
+
+3. Run Docker Compose:
+   ```bash
+   docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
+   ```
```

Comment:
hmmm these are docs written for a contributor no? personally found that docker was the easiest way for me to get everything up and running so that I could start adding a new panel

Reply:
Adding these as steps make it seem a bit like it's required, but I guess it's worth making a note that we have the docker compose. I personally find it a bit hard to work with, and like to just run reth outside of docker normally